pkgs/games/runescape3/default.nix

```diff
@@ -0,0 +1,109 @@
+{ stdenv, lib, buildFHSUserEnv, dpkg, glibc, gcc-unwrapped, autoPatchelfHook, fetchurl, wrapGAppsHook
+  , gnome2, xorg
+  , libSM, libXxf86vm, libX11, glib, pango, cairo, gtk2-x11, zlib, openssl
+  , libpulseaudio
+  , SDL2, xorg_sys_opengl, libGL
+}:
+let
+
+  version = "2.2.9";
+  pname = "runescape-launcher";
+
+  src = fetchurl {
+    url = "https://content.runescape.com/downloads/ubuntu/pool/non-free/r/${pname}/${pname}_${version}_amd64.deb";
+    sha256 = "0r5v1pwh0aas31b1d3pkrc8lqmqz9b4fml2b4kxmg5xzp677h271";
+  };
+
+  runescape = stdenv.mkDerivation {
+    name = pname;
+    system = "x86_64-linux";
```

Comment:
WHat is this?

Reply:
Could you clarify?

Reply:
Are you referencing line 19 or something else? Did I specify system incorrectly?

Reply:
Are you referencing line 19 or something else @SuperSandro2000 ? Did I specify system incorrectly?

Reply:
I believe you should remove the `system` here as it is already defined in `meta.platforms`.